hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/TokenUpdateNftsHandler.java

```diff
@@ -81,8 +89,25 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
         final var tokenStore = context.createStore(ReadableTokenStore.class);
         final var token = tokenStore.get(op.tokenOrElse(TokenID.DEFAULT));
```

Comment:
WE can use `getIfUsable` here to get token. It has some extra checks

Reply:
Updated this.